### ISSUE TITLE
A11y: update accessibility documentation for Design section of design.va.gov

### ISSUE THREAD

**Issue by k80bowman** [department-of-veterans-affairs/va.gov-team#37062]
## Description
We would like to update the accessibility guidance for two pages on design.va.gov. 

https://design.va.gov/design/
Update accessibility section to read:
>While this guide will always provide accessible samples and usage guidelines, it is still possible to create inaccessible designs using the components and utilities available information. Always remember that the audience using VA.gov may not be visual users, may experience color blindness or have difficulty handling low contrast ratios. Never use color alone as an affordance, always allow users to control automated content, and provide alternative navigation options.

https://design.va.gov/design/icons
Add the following to the top of the Accessibility section, above "When to use `role="img"`:
>When using a decorative icon, it shouldn’t be read out to screen-reader users because it doesn’t add to their understanding. Icons should be accompanied with text.
All interactive accessible icons must be keyboard accessible and touch accessible as well as clickable via a mouse.
